openassessment/workflow/models.py

```diff
@@ -335,12 +355,61 @@ def _get_steps(self):
             # If no steps exist for this AssessmentWorkflow, assume
             # peer -> self for backwards compatibility
             self.steps.add(
-                AssessmentWorkflowStep(name=self.STATUS.peer, order_num=0),
-                AssessmentWorkflowStep(name=self.STATUS.self, order_num=1)
+                AssessmentWorkflowStep(name=self.STATUS.peer, order_num=1),
```

Comment:
These order numbers seem brittle. What happens if the ``ORA2_ASSESSMENT_SCORE_PRIORITY`` setting changes the order? It seems that this should probably be based upon ``ASSESSMENT_SCORE_PRIORITY``. Do you know if there are tests that cover this logic?